docs/java-agent-injector.md

```diff
@@ -111,6 +117,73 @@ volumes:
     name: java-agent-configmap-volume
 ```
 
+### Use SwAgent CR to overlay default agent configuration
+
+The injector will read the SwAgent CR when pods creating.
+
+SwAgent CRD basic structure is like:
+
+```yaml
+apiVersion: operator.skywalking.apache.org/v1alpha1
+kind: SwAgent
+metadata:
+  name: swagent
+  namespace: skywalking-system
+spec:
+  containerMatcher: '.*'
+  selector:
+    app: demo
+  javaSidecar:
+    name: swagent
+    image: apache/skywalking-java-agent:8.8.0-java8
+    env:
+      - name: SW_LOGGING_LEVEL
+        value: "DEBUG"
+  sharedVolume:
+    name: "sky-agent"
+    mountPath: "/sky/agent"
+  swConfigMapVolume:
```

Comment:
From the `description`, this refers to the `default` configmap. Why do users have to set up it since it's the deault?